PWGHF/HFC/TableProducer/correlatorDsHadrons.cxx

```diff
@@ -614,6 +607,7 @@ struct HfCorrelatorDsHadrons {
                             particle.pt(),
                             particleAssoc.pt(),
                             poolBin);
+          entryDsHadronRecoInfo(1.968, true);
```

Comment:
No magic numbers

Reply:
Here I have a doubt: how can I access to a pdg mass of a particle?